code/game/turfs/turf.dm

```diff
@@ -169,6 +171,8 @@
 		air_master.remove_from_active(src)
 
 	var/turf/W = new path(src)
+	if(!defer_change)
+		W.AfterChange()
 
 	if(istype(W, /turf/simulated))
 		W:Assimilate_Air()
```

Comment:
Alright, I've finished: You can merge <a href=https://github.com/CrAzYPiLoT-SS13/Paradise/pull/29>this PR</a>, and this should fix:
* APCs being in the same area
* Pipes breaking on map load
* Atmos not correctly initializing

Best of luck!